types/express-ejs-layouts/express-ejs-layouts-tests.ts

```diff
@@ -0,0 +1,5 @@
+import * as expressEjsLayouts from 'express-ejs-layouts';
+
+function expressRequestHandlerTest() {
+    const requestHandler = expressEjsLayouts();
+}
```

Comment:
The test seems not right, you can use the code from the document as the test case, like:
```ts
import express = require('express');
import expressLayouts = require('express-ejs-layouts');

const app = express();
app.use(expressLayouts);
```

Reply:
Done